Merchant/App/Pin/PinController.swift

```diff
@@ -11,7 +11,7 @@ import BDCKit
 
 class PinController: PinViewController {
 
-    var pinMessageLabel: BDCLabel = {
+    let pinMessageLabel: BDCLabel = {
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possibly many more places than just these...